src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rusultInverse = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+    const chek = rusultInverse.hasOwnProperty([value]);
```

Comment:
please try to use the correct names chek => check

Reply:
Thanks!
my inattention((((